src/widgetastic_patternfly4/modal.py

```diff
@@ -1,30 +1,30 @@
 from widgetastic.exceptions import NoSuchElementException
 from widgetastic.utils import ParametrizedLocator
-from widgetastic.widget import Widget
+from widgetastic.widget import View
 from widgetastic.xpath import quote
 
 
 class ModalItemNotFound(Exception):
     pass
 
 
-class Modal(Widget):
+class Modal(View):
     """Represents the Patternfly Modal.
 
     https://www.patternfly.org/v4/documentation/react/components/modal
     """
 
     ROOT = ParametrizedLocator("{@locator}")
     BODY = ".//div[contains(@class, 'pf-c-modal-box__body')]"
-    FOOTER = ".//div[contains(@class, 'pf-c-modal-box__footer')]/child::node()"
+    FOOTER = ".//*[contains(@class, 'pf-c-modal-box__footer')]/child::node()"
```

Comment:
Here `div` is replaced with `footer` tag;  As `pf-c-modal-box_footer` class attribute provides strong support so just replaced with *.